config/default.yml

```diff
@@ -2545,6 +2545,12 @@ Style/BlockDelimiters:
   # the `line_count_based` style.
   BracesRequiredMethods: []
 
+Style/BooleanKeywordArgument:
+  Description: 'Use keyword arguments when defining method with boolean argument.'
+  StyleGuide: '#boolean-keyword-arguments'
+  Enabled: pending
```

Comment:
I think that this cop is unsafe, just like `Style/OptionalArguments` cop.

Reply:
I'm curious why `Style/OptionalArguments` is "unsafe", or this one?

For this one, my only concern is if there is a call to `super`, or `super(that_argument)`, then maybe there should be no offense since this method is just copying the interface it inherits. Maybe there should not be an offense then; the base method will be flagged probably. What other cases are unsafe?

Reply:
I think it means a breaking change to users if the API signature is a published interface.
https://martinfowler.com/bliki/PublishedInterface.html

Reply:
Made it unsafe.